### DISCUSSION TITLE
Welcome to JDBC_Generics_ShopOnline Discussions!

### DISCUSSION THREAD

**Discussion by HoseinRezaeeM** [HoseinRezaeeM/JDBC_Generics_ShopOnline#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SinaRahimii75** [comment]
very Good

**Comment by SinaRahimii75** [comment]
best idea generic - jdbc

**Comment by HoseinRezaeeM** [comment]
yes sina
